addons/knowledge/static/src/js/field_html.js

```diff
@@ -0,0 +1,85 @@
+/** @odoo-module */
```

Comment:
don't use the `static/src/js` and `static/src/scss` structure. now, files should be located by concern: `static/src/something` which contains js/css/xml related to the `something` concern

Reply:
Todo post-merge